### PR TITLE
fix: changed assign to field as typeable input

### LINKED ISSUE #8412: [CAL-1542] Make "Assign to" type-ahead
turn this select:
CleanShot 2023-04-20 at 18 46 51@2x

into a typeable input

<sub>[CAL-1542](https://linear.app/calcom/issue/CAL-1542/make-assign-to-type-ahead)</sub>

### PR DESCRIPTION
**Pull request by joe-shajan**
## What does this PR do?

Fixes #8412

Changed assign_to field as a simple typable input.

**After**
assign to dark

So after this `ChildrenEventTypeSelect` file is not used anymore right? but when I tried to delete it. I noticed that it's also used in `event-types/[type]/index.tsx` Can I remove it from there also? Or is there any other use for it?

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by joe-shajan** [comment]
e2e test is failing is it because I changed the select input to normal typable input? @PeerRich

**Review by hariombalhara — CHANGES_REQUESTED** [review]
I think what @PeerRich meant here was to enable search within the select. What I see is it has become just a text field now.

**Comment by hariombalhara** [comment]
Select supports it already. I think it's disabled there not sure why

**Comment by joe-shajan** [comment]
> Select supports it already. I think it's disabled there not sure why

Ok thanks, will check that out.

File: packages/prisma/.env
Comment (joe-shajan) [inline packages/prisma/.env:1]: This happened by mistake. I'm not able to fix it. Why is this compared since the content are same?

**Comment by joe-shajan** [comment]
Hi, @hariombalhara searchable for select was set to false. So I removed that line because the select compnent is searchable by default.

https://user-images.githubusercontent.com/69904519/235213736-0a41db5c-041f-4670-bb00-da89a54ef5b3.mp4

File: packages/prisma/.env
Comment (joe-shajan) [reply packages/prisma/.env:1]: Can you help me with this? @hariombalhara

**Comment by Udit-takkar** [comment]
@joe-shajan just create a new branch and raise a new PR with packages/features/eventtypes/components/ChildrenEventTypeSelect.tsx changes

**Comment by joe-shajan** [comment]
> @joe-shajan just create a new branch and raise a new PR with packages/features/eventtypes/components/ChildrenEventTypeSelect.tsx changes

Sure will do that 👍

**Comment by joe-shajan** [comment]
Closing this pull request and raised a new one #8637

**Pull request closed without merging**